codebase/codebase.go

```diff
@@ -37,22 +40,41 @@ func (c *CodebaseContent) ToMap() map[string]interface{} {
 	res[BranchKey] = c.Branch
 	res[FileNameKey] = c.FileName
 	res[LineNumberKey] = c.LineNumber
+	res[CodebaseIDKey] = c.CodebaseID
 	return res
 }
 
+// validateURL makes sure Repo is valid GIT URL
+func validateURL(repo string) (string, bool) {
+	validPrefixes := []string{"https://github.com/", "git@github.com:"}
```

Comment:
thank you , now using this and added respective test in here https://github.com/almighty/almighty-core/pull/1241/commits/23079dffacae9a88455ae693fa0e2d4786629b63